packages/calypso-products/src/types.ts

```diff
@@ -75,14 +75,18 @@ export type SelectorProductFeaturesItem = {
 	isDifferentiator?: boolean;
 };
 
+export interface JetpackTag {
+	tag: string;
+	translation: TranslateResult;
```

Comment:
Instead of calling this `translation`, it's actually a label/name. Can we name it likewise?